app/jobs/regular/emit_web_hook_event.rb

```diff
@@ -110,6 +110,15 @@ def web_hook_request(args, web_hook)
           web_hook_event_id: web_hook_event.id,
           event_type: args[:event_type]
         }, user_ids: User.human_users.staff.pluck(:id))
+
+        if SiteSetting.retry_web_hook_events? && response.status != 200
+          args[:retry_count] ||= 0
+          return if args[:retry_count] > 4
+
+          delays = [0, 1, 5, 15, 60]
```

Comment:
@SamSaffron  Are you OK if we go with an exponential backoff instead? `2 ** retries`? I'll prefer that instead of having to assign a constant for the retries minutes.

Reply:
@vinothkannans Hmm should we retry immediately the first time it fails? I feel like the first retry should include the delay.